app/views/transaction-detail/index.njk

```diff
@@ -12,7 +12,7 @@
   {{ super() }}
   {% set defaultBackLink %}{{formatAccountPathsFor(routes.account.transactions.index, currentGatewayAccount.external_id)}}{% if indexFilters %}?{{indexFilters}}{% endif %}{% endset %}
```

Comment:
Should this also factor in all service transactions test URL ?

Reply:
I don't think this has anything to do with this PR